### ISSUE TITLE
how to re-implement???

### ISSUE THREAD

**Issue by Johnson-yue** [MarziEd/SubSpace-Capsule-Network#1]
thanks your sharing ,but how to use it??  any ReadMe update??

**Comment by orbennatan** [comment]
Yes. Please. ReadMe update. I am working to reverse engineer how to run the commands, but it is very time consuming. 
Please help.
